### PR TITLE
docs: update disclosure policy

### PR DESCRIPTION
**Pull request by CjS77**
The disclosure and bug bounty policy is updated to account for:
* HackerOne bounty programme
* New Github private disclosures
* Updates bounty reward values

Old and invalid information has been removed.

Funky graphic!

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -19,6 +28,10 @@ This policy does not apply to:
 
 * Archived repositories;
 * End-user documentation and educational or "help" materials (e.g., [Tari Labs University](https://tlu.tarilabs.com/));
+* Example or test code.
+* Proof-of-concept or demonstration applications.
+* Code marked `NOT READY FOR PRODUCTION`, unless the code is demonstrably used in production applications, and those
```
Comment (AaronFeickert) [inline SECURITY.md]:
```suggestion
* Code or repositories indicated not to be suitable for production, unless the code is demonstrably used in production applications, and those 
```
Presumably the exact wording is not required?

Comment (AaronFeickert) [reply SECURITY.md]: For example, the new [Triptych](https://github.com/tari-project/triptych) repository is experimental and would be out of scope, but has a differently-worded [scary warning](https://github.com/tari-project/triptych?tab=readme-ov-file#warning).

Comment (CjS77) [reply SECURITY.md]: Correct. Exact wording is not necessary

Comment (CjS77) [reply SECURITY.md]: I've reworded to make this explicit.

**Review by SWvheerden — APPROVED** [review]
LGTM

**Pull request merged**